### PR TITLE
Add CMakelists

### PR DESCRIPTION
**Pull request by 0xc0170**
Following https://github.com/ARMmbed/mbed-os/pull/13566

Adding CMakelists to our examples

I still need to fix travis, circleCI should work.

Please note, this is still in development (for up to date status, see mbed-os/cmake/README.md)

### CODE REVIEW

**Comment by 0xc0170** [comment]
I updated travis to use master branch. It will fail now but once this gets to master, it should be green (will retest via new PR to make sure travis config is correct).

**Comment by 0xc0170** [comment]
@hugueskamba Please review. I can squash these into one commit, as the changes are simple and nothing important in the history to be kept.

**Comment by hugueskamba** [comment]
> @hugueskamba Please review. I can squash these into one commit, as the changes are simple and nothing important in the history to be kept.

Thank you and yes please squash.

**Comment by 0xc0170** [comment]
Squashed

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13)
```
[the remark this answers is not in the document]
Comment (0xc0170) [reply CMakeLists.txt:4]: I've noticed as well, will update all

[the remark this answers is not in the document]
Comment (0xc0170) [reply CMakeLists.txt:4]: updated

**Review by hugueskamba — APPROVED** [review]

**Comment by 0xc0170** [comment]
> Following ARMmbed/mbed-os#13566

It was merged.

I fixed travis to use master branch, once CI is all green, this should get in.

**Comment by 0xc0170** [comment]
Fixing the memmap requirements, we will need to figure out the python deps and how to use the scripts.

**Review by hugueskamba — APPROVED** [review]

**Pull request merged**